src/region/RegionCreator.tsx

```diff
@@ -50,7 +51,14 @@ export default function RegionCreator({ className, onAbort, onStart, onStop }: P
         const { current: x2 } = positionX2Ref;
         const { current: y2 } = positionY2Ref;
 
-        if (!creatorEl || x1 === null || x2 === null || y1 === null || y2 === null) {
+        if (
```

Comment:
Is this needed? It seems like what we want is to suppress the creation of a really small region if the intention of the user is to click on the document -- not necessarily to suppress the creation of a really small region always.